testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -1,8 +1,10 @@
 # Copyright (c) 2019-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
+#
 # This feature is a dependency for:
 # - features/secondary/min_monitoring.feature : If this feature fails could let monitoring feature disabled for SLE minion
-# - features/secondary/min_centos_monitoring.feature : If this feature fails could let monitoring feature disabled for CentOS minion
+# - features/secondary/min_rhlike_monitoring.feature : If this feature fails could let monitoring feature disabled for RedHat-like minion
```

Comment:
```suggestion
# - features/secondary/min_rhlike_monitoring.feature : If this feature fails, it could let the monitoring feature disabled for RedHat-like minion
```

Reply:
Same for line 5 but you did not touch that one.

Reply:
Thank you, fixing.